test/spec/parserSpec.js

```diff
@@ -53,11 +53,32 @@ var xml = (function() {
   </codesConverter>
 </deployment>`;
 })();
-    
+
+var tags = (function() {
+    var tags = {};
+    tags.component = "component";
+    tags.name = "name";
+    tags.id = "id";
+    tags.stateMachine = "stateMachine";
+    tags.publish = "publish";
+    tags.componentCode = "componentCode";
+    tags.stateMachineCode = "stateMachineCode";
+    tags.eventCode = "eventCode";
+    tags.event = "event";
+    tags.topic = "topic";
+    tags.eventCode = "eventCode";
+    tags.subscribe = "subscribe";
+    return tags;
+})();    
+
+
+
 define(["parser"], function (Parser) {
+    
+    var parser = new Parser();
+    parser.parse(xml, tags);
 
     describe("Test Parser module", function () {
-        var parser = new Parser(xml);
 
         it("Test getCodes with correct basic cases", function () {
```

Comment:
Better description (with something like it should...)
"should get the right codes when calling getCodes with existing component name and stjatemachine name